NIPTool/server/templates/statistics.html

```diff
@@ -0,0 +1,133 @@
+{% extends 'layout/layout.html' %}
+{% block body %}
+{% include 'layout/navigation.html' %}
+<div class="container-fluid">
+    <div class="jumbotron vertical-center">
+        <div class="row">
+            <div class="col-lg-12">
+                <h1 class="page-header"><strong>Quality Control/QC </strong></h1>
+            </div>
+        </div>
+    </div>
+    <div class="row">
+    </div>
+    <ul class="nav nav-tabs" id="myTab">
+        {% for plot in box_plots %}
+        <li><a href="#{{plot}}" data-toggle="tab">{{plot}}</a></li>
+        {% endfor %}
+        {% for plot in scatter_plots %}
+        <li><a href="#{{plot}}" data-toggle="tab">{{plot}}</a></li>
+        {% endfor %}
+    </ul>
+    <div class="tab-content">
+        {% for plot in box_plots %}
+        <div class="tab-pane fade in active" id='{{plot}}'>
+            <div id="{{plot}}" style="min-width:1000px; height:600px; "></div>
+        </div>
+        {% endfor %}
+        {% for plot in scatter_plots %}
+        <div class="tab-pane fade in active" id='{{plot}}'>
+            <div id="{{plot}}" style="min-width:1000px; height:600px; "></div>
+        </div>
+        {% endfor %}
+    </div>
+</div>
+{% endblock %}
+
+{% block scripts %}
+
+<script>
+
+    ////////////////// Box Plots //////////////////
+    {% for plot_name in box_plots %}
+    var XY_plot = document.getElementById('')
```

Comment:
I cannot find a use for this variable, is it used anywhere? Why passing an empty string to it? It will be always null.

Reply:
👍